defaults/main.yml

```diff
@@ -59,6 +59,7 @@ nginx_vhosts: []
 nginx_upstreams: []
 # - name: myapp1
 #   strategy: "ip_hash" # "least_conn", etc.
+#   keepalive: 16
```

Comment:
Can you add, at the end of the line, `# optional` (just to make it obvious in the defaults).